docs/csharp/whats-new/csharp-9.md

```diff
@@ -12,20 +12,20 @@ C# 9.0 adds the following features and enhancements to the C# language:
 - [Top-level statements](#top-level-statements)
 - [Pattern matching enhancements](#pattern-matching-enhancements)
 - [Performance and interop](#performance-and-interop)
-  - Native sized integers
-  - Function pointers
-  - Suppress emitting localsinit flag
+  - [Native sized integers](../../../../../../csharplang/blob/main/proposals/csharp-9.0/native-integers.md)
```

Comment:
```suggestion
  - [Native sized integers](~/_csharplang/proposals/csharp-9.0/native-integers.md)
```

Reply:
Thanks for your help but I have already tried the tilde character for the root before sent the commit. I mean, your that url -and the others- generates https://github.com/dotnet/docs/blob/main/docs/csharp/whats-new/~/_csharplang/proposals/csharp-9.0/native-integers.md and this link is returning HTTP status code 404 normally. And also you can try `chsarp` instead `_chsarp` path. The result is the same.

Reply:
@g-u-r-k-a-n My suggestion will produce 404 when viewing in GitHub, but will work on the actual docs site. Your current approach is the opposite. Using `~/_csharplang` is specific to how building the repository works, and is defined in the configuration file:

https://github.com/dotnet/docs/blob/b5a8fecd1c24be9db5267395e5f6d0c1b514e98f/.openpublishing.publish.config.json#L48-L53

Reply:
Aww, just started browsing the documents today. I didn't know that config file. Then I will approve your suggestions soon.